doc/guides/gps_setup.md

```diff
@@ -0,0 +1,49 @@
+---
+module: [kind=guide] gps_setup
+---
+
+# Setting up GPS
+The @{gps} API allows computers and turtles to find their current position using wireless modems.
+
+In order to use GPS, you'll need to set up multiple GPS hosts. These are computers running the special gps host program, which tell other computers the host's position. Several hosts running together are known as a GPS constellation.
+
+Before GPS can be used in a dimension (e.g. the Nether) a GPS host constellation needs to be set up. Traditionally this is done near the max build height for maximum range, but ender modems easily beats a normal modem's max range at any height.
```

Comment:
The first bit of this kinda duplicates the previous paragraph now.

I think I'd probably drop this paragraph (and the one below) and then mention these considerations in the "Building a GPS constellation" section.